packages/reactive-element/src/reactive-element.ts

```diff
@@ -1421,12 +1421,11 @@ export abstract class ReactiveElement
         .elementProperties;
       if (elementProperties.size > 0) {
         for (const [p, options] of elementProperties) {
-          if (
-            options.wrapped === true &&
-            !this._$changedProperties.has(p) &&
-            this[p as keyof this] !== undefined
-          ) {
-            this._$changeProperty(p, this[p as keyof this], options);
+          if (options.wrapped === true && this[p as keyof this] !== undefined) {
+            // Note, deleting an existing value here ensures
+            // the initial old value is seeded as `undefined`
```

Comment:
When would `_$changedProperties` have something else in it?

Reply:
Hmm, I would think we would return the getter's default value for the old value in that case, but I see how we can't do that if we want the initial value of an accessor to have undefined as its previous value and can't identify an initializer.